source/gui/guiHelper.py

```diff
@@ -301,3 +302,7 @@ def addDialogDismissButtons(self, buttons):
 		self.dialogDismissButtonsAdded = True
 		return buttons
 
+class SIPABCMeta(wx.siplib.wrappertype, ABCMeta):

```

Comment:
Why does this need to be used with wx subclasses?

Reply:
WX Python 3 used swig to wrap WX C++ classes, WX Python Phoenix uses sip.

```
>>> type(wx.Dialog)
<type 'sip.wrappertype'>
```

If we don't create SIPABCMeta and just assign ABCMeta as meta class to SettingsDialog or SettingsPanel, NVDA will fail to start due to a meta class conflict, since ABCMeta and wx.siplib.wrappertype are unrelated to each other. You can't assign multiple meta classes to a class.